### PR TITLE
Add tokens for money input  new design

### PR DESCRIPTION
**Pull request by Rhotimee**
#### Summary
Implement new design for money input component

### CODE REVIEW

File: packages/components/inputs/money-input/src/money-input.styles.ts [hunk 1]
```diff
@@ -11,10 +11,14 @@ const getCurrencyLabelStyles = () => css`
   border-bottom-left-radius: ${designTokens.borderRadiusForInput};
   border: 1px ${designTokens.borderColorForInputWhenDisabled} solid;
   border-right: 0;
-  padding: 0 ${designTokens.spacing20};
+  padding: ${designTokens.paddingForMoneyInputCurrencyDropdown};
   align-items: center;
   font-size: ${designTokens.fontSizeForInput};
   box-sizing: border-box;
+
+  &:hover {
```
Comment (kark) [inline packages/components/inputs/money-input/src/money-input.styles.ts:19]: Let me pin it here, this comment is not directly related to this line.

I'm not sure if this if OK, but I noticed that when `isReadOnly` is truthy:
the border of `<CurrencyLabel>` is 1px wide
Screenshot 2023-02-06 at 12 22 06

**Review by ddouglasz — COMMENTED** [review]
Good outcome 💪. Every other thing looks great! I only just noticed that the new design `value focused` only highlights the border of the text input value section and not the currency dropdown section 
expected:
image

currently:
image

File: packages/components/inputs/money-input/src/money-input.tsx [hunk 2]
```diff
@@ -825,6 +832,11 @@ const MoneyInput = (props: TMoneyInputProps) => {
                 css`
                   padding-right: ${designTokens.spacing40};
```
Comment (kark) [inline packages/components/inputs/money-input/src/money-input.tsx:833]: It seems to me the high precision badge styles should also be updated:
Screenshot 2023-02-06 at 12 52 07
to match the designs
Screenshot 2023-02-06 at 12 56 45

**Review by kark — COMMENTED** [review]
Great work 💪

File: packages/components/inputs/money-input/src/money-input.styles.ts (hunk 1 above)
Comment (Rhotimee) [reply packages/components/inputs/money-input/src/money-input.styles.ts:19]: Updated [11712c6](https://github.com/commercetools/ui-kit/pull/2397/commits/11712c63c56868b44f14f3e3f2e0bfd229f7aa4f)

File: packages/components/inputs/money-input/src/money-input.tsx (hunk 2 above)
Comment (Rhotimee) [reply packages/components/inputs/money-input/src/money-input.tsx:842]: Updated [11712c6](https://github.com/commercetools/ui-kit/pull/2397/commits/11712c63c56868b44f14f3e3f2e0bfd229f7aa4f)

File: design-system/materials/internals/definition.yaml [hunk 3]
```diff
@@ -761,6 +767,13 @@ decisionGroupsByTheme:
         width-for-avatar-as-medium:
           choice: '48px'
 
+    minwidths:
+      label: Width
+      prefix: width
```
[the remark this answers is not in the document]
Comment (Rhotimee) [reply design-system/materials/internals/definition.yaml:774]: Updated [11712c6](https://github.com/commercetools/ui-kit/pull/2397/commits/11712c63c56868b44f14f3e3f2e0bfd229f7aa4f)

**Comment by CarlosCortizasCT** [comment]
Nice job so far 🙇 

I noticed that the margin right we use for the high precision badge icon is `16px` but there are `12px` [in the design](https://5jpcut.axshare.com/?id=5b1b5dd3-2598-49b8-8fd9-5e16aa1e9411&g=5).

Also, some hovered and read only states coloes and borders are not like in the design, but maybe that's something you are syncing with Douglas to handle it.

**Comment by ddouglasz** [comment]
> Nice job so far 🙇
> 
> I noticed that the margin right we use for the high precision badge icon is `16px` but there are `12px` [in the design](https://5jpcut.axshare.com/?id=5b1b5dd3-2598-49b8-8fd9-5e16aa1e9411&g=5).
> 
> Also, some hovered and read only states coloes and borders are not like in the design, but maybe that's something you are syncing with Douglas to handle it.

Yes, the hover for inputs and border for inputs when readOnly are handled in this PR https://github.com/commercetools/ui-kit/pull/2412

File: design-system/materials/internals/definition.yaml [hunk 4]
```diff
@@ -761,6 +769,13 @@ decisionGroupsByTheme:
         width-for-avatar-as-medium:
           choice: '48px'
 
+    minwidths:
```
[the remark this answers is not in the document]
Comment (Rhotimee) [reply design-system/materials/internals/definition.yaml]: updated [3718bb5](https://github.com/commercetools/ui-kit/pull/2397/commits/3718bb5b966d8dd7c795c97992bde76d1783d430)

File: design-system/materials/internals/definition.yaml [hunk 5]
```diff
@@ -1417,6 +1438,13 @@ decisionGroupsByTheme:
         width-for-avatar-as-medium:
           choice: '40px'
 
+    minwidths:
+      label: Width
+      prefix: width
```
[the remark this answers is not in the document]
Comment (Rhotimee) [reply design-system/materials/internals/definition.yaml]: updated [3718bb5](https://github.com/commercetools/ui-kit/pull/2397/commits/3718bb5b966d8dd7c795c97992bde76d1783d430)

**Comment by Rhotimee** [comment]
As @ddouglasz mentioned. Some of the updates for the new design are addressed in this [PR](https://github.com/commercetools/ui-kit/pull/2412)

File: design-system/materials/internals/definition.yaml (hunk 5 above)
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply design-system/materials/internals/definition.yaml]: Please, remember to also update the `label` and `prefix` (also in the `default` theme block)

**Review by ddouglasz — APPROVED** [review]
Looks good 🙌

File: design-system/materials/internals/definition.yaml (hunk 5 above)
[the remark this answers is not in the document]
Comment (Rhotimee) [reply design-system/materials/internals/definition.yaml]: Thanks. I missed that. 
Updated [a92848c](https://github.com/commercetools/ui-kit/pull/2397/commits/a92848c1066105cb970ecd552364137cda3aab42) [67e8bd7](https://github.com/commercetools/ui-kit/pull/2397/commits/67e8bd7f89df762288afa26daf5868e85dd0b297)

**Review by CarlosCortizasCT — APPROVED** [review]
Cool 😎

**Review by kark — APPROVED** [review]
🙌
